lms/templates/admin/organization.html.jinja2

```diff
@@ -0,0 +1,34 @@
+{% import "macros.jinja" as macros %}
+
+{% extends "lms:templates/admin/base.html.jinja2" %}
+{% set public_id = org.public_id(request.region) %}
+{% block header %}
+    Organization {{org.name}} / {{ public_id }}
```

Comment:
This is a bit eyeboggling in practice I think as it ends up with things like:

> Organization Blackboard, Inc! / us.lms.org.fY9f8UQORZCPopiRNGkK1w

As most of this detail is replicated below I wonder if we want to use this to display something distinct. For example we don't expose the primary key anywhere, and this is what we might use for most queries. This would like up with other pages, so the title would be:

> Organization 123